src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return Colors.values()[(int)(Math.random() * (Colors.values().length + 1))].name();
```

Comment:
Use Random class for random value.
Like:
![image](https://user-images.githubusercontent.com/12864128/191293508-aa3c3a2e-4086-45e1-98df-77bc0f1ac543.png)